### PR TITLE
Add lendingAccount & fundingWallet endpoints implementations

### LINKED ISSUE #482: add 2 new endpoints to be able display a global balance
Hello to contributers of this package

First of all thanks.
I am mainly using this api for market panic sell
or displaying my whole balance calculated in both € and btc
but i have to "manually" insert these 2 new endpoints to take into account locked savings and binance card wallet. Could you add them in a new version ? 

```
,
    lendingAccount: function lendingAccount(payload) {
      return privCall('/sapi/v1/lending/union/account', payload);
    },
    binanceCard: function binanceCard(payload) {
      return privCall('/sapi/v1/asset/get-funding-asset', payload, 'POST');
    },
```

### PR DESCRIPTION
**Pull request by vsly-ru**
Closes #482 by implementing
- [Lending Account (USER_DATA)](https://binance-docs.github.io/apidocs/spot/en/#lending-account-user_data) endpoint
- [Funding Wallet (USER_DATA)](https://binance-docs.github.io/apidocs/spot/en/#funding-wallet-user_data) endpoint
- Corresponding data types for TypeScript

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -89,6 +89,34 @@ declare module 'binance-api-node' {
...
+    totalAmountInUSDT: string
+    totalFixedAmountInBTC: string
+    totalFixedAmountInUSDT: string
+    totalFlexibleInBTC: string
+    totalFlexibleInUSDT: string
+  }
+
+  export interface FundingWallet {
+    asset: string
+    free: string    // avalible balance
```
Comment (balthazar) [inline index.d.ts:113]: little comment typo

**Review by balthazar — CHANGES_REQUESTED** [review]
Do you mind also adding the two methods to the readme?

File: index.d.ts (hunk 1 above)
Comment (vsly-ru) [reply index.d.ts:113]: > little comment typo

haha, it was just copied from the official documentation (with the typo)

**Comment by balthazar** [comment]
Thank you!

**Pull request merged**